compiler/rustc_passes/src/check_attr.rs

```diff
@@ -2105,7 +2105,7 @@ impl CheckAttrVisitor<'_> {
     }
 
     fn check_macro_export(&self, hir_id: HirId, attr: &Attribute, target: Target) {
-        if target != Target::MacroDef {
+        if target != Target::MacroDef  {
```

Comment:
rustfmt doesn't support if let chains so it doesn't format this, but this extra space should be removed.

Reply:
woops, that managed to sneak past me. Thanks for catching it :) fixed in bdf4e3d and rebased the branch